RecoEgamma/EgammaElectronProducers/plugins/GsfElectronCoreBaseProducer.h

```diff
@@ -27,10 +27,10 @@ namespace edm {
 
 class GsfElectronCoreBaseProducer : public edm::stream::EDProducer<> {
 public:
-  static void fillDescription(edm::ParameterSetDescription&);
+  static edm::ParameterSetDescription makeBaseDescription(std::string const& gsfPfRecTracks,
+                                                          std::string const& gsfTracks);
```

Comment:
FWIW, "pfTrackElec" and "electronGsfTracks" can be default values for these arguments (so that it does not have to be repeated in 3 places)